docs/core/animations.md

```diff
@@ -73,6 +73,16 @@ We can animate numeric attributes as well. For example, we can animate the inten
 </a-light>
 ```
 
+### Color Attributes
+
+We can animate color attributes as well. For example, we can animate a box from white to red.
+
+```html
+<a-entity id="blushing-cube" geometry="primitive: box">
+  <a-animation attribute="color" begin="white" to="red" dur="1000"></a-animation>
```

Comment:
Should be `attribute="material.color"`

`begin="white"` should be `from="white"`